pkg/kubectl/cmd/get/get.go

```diff
@@ -118,6 +118,9 @@ var (
 		# Return only the phase value of the specified pod.
 		kubectl get -o template pod/web-pod-13je7 --template={{.status.phase}}
 
+		# List resource information in custom columns.
+		kubectl get pod test-pod -o custom-columns=CONTAINER:.spec.containers[-2:].name,IMAGE:.spec.containers[-2:].image
```

Comment:
this seems a little verbose / edge-case, but will leave it to the maintainers to comment.

Reply:
Hi，I want to show this edge–case to let people, especially new users， to grab it thoroughly

Reply:
a custom-columns example is fine, I'd probably choose a simpler example jsonpath